src/SymbolCollector.Android.Library/Host.cs

```diff
@@ -50,22 +51,22 @@ public static IHost Init(string dsn)
                             break;
                     }
 
-                    try
-                    {
-                        // TODO Add to Sentry.Xamarin
-#pragma warning disable 618
-                        @event.Contexts.Device.Architecture = Build.CpuAbi;
-#pragma warning restore 618
-                        // TODO: Same as Brand though?
-                        @event.Contexts.Device.Manufacturer = Build.Manufacturer;
-
-                        // Auto tag at least on error events:
-                        // @event.SetTag("device", Build.Device ?? "?");
-                    }
-                    catch
-                    {
-                        // Capture the event without these values
-                    }
+//                     try
+//                     {
+//                         // TODO Add to Sentry.Xamarin
+// #pragma warning disable 618
+//                         @event.Contexts.Device.Architecture = Build.CpuAbi;
```

Comment:
If we set this now OOTB, we can drop it from here.

Reply:
I'll merge the dead code in for now and after a run on appcenter we'll see if we delete this or what